server/src/config.ts

```diff
@@ -256,8 +256,8 @@ export const defaults = Object.freeze<SystemConfig>({
       modelName: 'ViT-B-32__openai',
     },
     duplicateDetection: {
-      enabled: false,
-      maxDistance: 0.03,
+      enabled: true,
+      maxDistance: 0.0155,
```

Comment:
This seems oddly specific

Reply:
golden ratio

Reply:
I have a few datasets with closely similar assets and this value returns the best grouping